src/components/manage/Sidebar/ObjectBrowser.jsx

```diff
@@ -74,9 +74,11 @@ const withObjectBrowser = (WrappedComponent) =>
     closeObjectBrowser = () => this.setState({ isObjectBrowserOpen: false });
 
     render() {
-      let contextURL = this.props.pathname;
-      if (this.props.pathname?.endsWith('edit')) {
-        contextURL = getParentURL(this.props.pathname);
+      let contextURL = this.props.pathname
```

Comment:
I think that I should remove `this.props.pathname` since we already extracting the contexturl from location. what you all think??

Reply:
Could you be more specific? I don't see why we need to change this.